website/src/docs/react.md

```diff
@@ -19,60 +19,56 @@ Install from NPM:
 npm install @uppy/react
 ```
 
-## CSS
-
-Make sure to also include the necessary CSS files for each Uppy React component you are using. Follow links for individual components docs below for details on that.
-
 ## Usage
 
 The components can be used with either [React][] or API-compatible alternatives such as [Preact][].
 
 Instead of adding a UI plugin to an Uppy instance with `.use()`, the Uppy instance can be passed into components as an `uppy` prop.
 All other props are passed as options to the plugin.
 
+You have to pass the IDs of your `use`d plugins to the `plugins` array props so Dashboard knows it needs to render them.
```

Comment:
I don't think this reasoning is required. 

- It is possible to make it work in React Strict Mode, we just chose not to do it.
- If a new user comes along, they never had to call `close` in general, so writing that you don't need to call it is unnecessary

I'll write a warning about using uppy in the render loop though 👍

Reply:
Did we really find a way that also lets us close uppy on unmount though? I tried to implement such a hook and document it but I failed because in strict mode we don’t know when a component is unmounted. Unmount is not a concept anymore with strict mode. If there is a way of doing it I think we should give an example in our docs because it is really *not* trivial!

as for close, maybe we dhould instead write it under uppy’s close-documentation (that people normally don’t need to call that function and it will make the uppy unusable after it’s closed)?

Reply:
I don't think we want to provide an example because it would only encourage what we want to discourage? Regarding close, I think we might be confused because we worked a long time hiding it in the hook, but note that people never had this confusion. It was hidden in the hook and we never encouraged in the docs to always call it, so I don't see why we now need to warn to not do it.

Reply:
I still think it makes sense to inform users that they should use a single uppy instance, so they don’t waste times trying to inplement hooks and Uppy as React state themselves. 

I think we should document close somewhere. As in what it does and when to use it, but we dont need to document it here under «react»

Reply:
> I still think it makes sense to inform users that they should use a single uppy instance, so they don’t waste times trying to inplement hooks and Uppy as React state themselves.

But we have that:

> * The Uppy instance should **not** live inside the component but outside of it.

What else do we need?

Reply:
i missed that one sry!